### DISCUSSION TITLE
Is this like minimongo from meteor but for postgres?

### DISCUSSION THREAD

**Discussion by TomFreudenberg** [electric-sql/electric#1006] [Questions] [answered]
Hi at all,

nice work and thanks for open source at first.

To identify correctly what electric does, it seems to me as if it was anounced like Meteors real time sync feature with DDL and minimongo years ago.

Is that a correct interpretation?

Second, while using Hasura is there anything in better for us when using electric?

Thanks for help and feedback.
Tom

**Comment by thruflo** [comment, marked as answer]
The closest analogy from the Mongo world would be "Realm for Postgres".

The key difference to realtime sync is the local-first architecture and offline support. If you dropped Hasura onto Postgres and used optimistic writes to get local-first with GraphQL, then the key difference is that we solve the concurrency issues and provide a conflict-free and rollback-free development model.

So no rollback handlers and no distributed systems complexity leaking back into your app layer.

**Reply by TomFreudenberg** [reply]
Thanks James @thruflo

Not sure if you know Meteorjs and minimongo, it had the same target and solution - local first. The technology behind surely is quite different and not comparable.

For Hasura that means - no Hasura anymore, when we want to use local-first by electric and get the benefits out from that, right?

**Comment by KyleAMathews** [comment]
👋 we've been working the last month on a rebuild of the Electric server over at a temporary repo https://github.com/electric-sql/electric-next/

You can read more about why we made the decision at https://next.electric-sql.com/about

We're really excited about all the new possibilities the new server brings and we hope you'll check it out soon and give us your feedback.

We're now moving the temporary repo back here. As part of that migration we're closing all the old issues, PRs, and discussions. We really appreciate you taking the time to start this discussion! If it's still relevant in the new system, please reopen.
